config/crd/bases/policies.kubewarden.io_admissionpolicies.yaml

```diff
@@ -348,6 +350,340 @@ spec:
     storage: true
     subresources:
       status: {}
+  - additionalPrinterColumns:
```

Comment:
Like this whole section, it didn't exist before. However this doesn't seem to be dangerous

Reply:
yes it was already [there](https://github.com/kubewarden/kubewarden-controller/blob/main/config/crd/bases/policies.kubewarden.io_admissionpolicies.yaml#L18) 
Now it is twice, one for v1alpha2 and another for v1